<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Coordinates</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="x_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="getdata">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Coordinates</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="y_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="z_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="marker1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Marker 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="marker2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Marker 2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="y_input_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="z_input_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="x_input_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="marker3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Marker 3</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="y_input_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="z_input_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="x_input_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="xPos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="yPos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="zPos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="y_entry">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="z_entry">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="entry">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Entry</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="x_entry">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="y_cannula">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="x_cannula">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="z_cannula">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cannula">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cannula</string>
    </property>
   </widget>
   <widget class="QLabel" name="zPos_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="yPos_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="xPos_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listMarkers">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listKeypoints">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="z_target1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="target1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Target 1</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="x_target1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="y_target1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate_angle">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>400</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Angle</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>load</sender>
   <signal>clicked()</signal>
   <receiver>x_input</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>getdata</sender>
   <signal>clicked()</signal>
   <receiver>listMarkers</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>getdata</sender>
   <signal>clicked()</signal>
   <receiver>listKeypoints</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Count()</slot>
 </slots>
</ui>
